src/SOS/SOS.Hosting/SOSHost.cs

```diff
@@ -171,7 +171,7 @@ internal unsafe int GetExecutingProcessorType(
                 Architecture.X86 => IMAGE_FILE_MACHINE.I386,
                 Architecture.Arm => IMAGE_FILE_MACHINE.ARMNT,
                 Architecture.Arm64 => IMAGE_FILE_MACHINE.ARM64,
-                (Architecture)6 /* Architecture.LoongArch64 */=> IMAGE_FILE_MACHINE.LOONGARCH64,
+                Architecture.LoongArch64 => IMAGE_FILE_MACHINE.LOONGARCH64,
```

Comment:
This isn't going to work since we build and have to build with the .NET 6.0 SDK.